tests/blas/nodes/axpy_test.py

```diff
@@ -314,7 +334,7 @@ def intel_fpga_graph(veclen, precision, vendor, testCase="0"):
     return test_sdfg.compile()
 
 
-def _test_fpga(type, vendor):
+def test_fpga(type, vendor):
```

Comment:
This is not good for pytest. It finds these tests and fails to run. We have to ensure that it is either in a valid pytest format (with `@pytest.mark.parametrize`) or not start with `test_`. This is why I changed it to `_test_fpga`.